src/routes/index.tsx

```diff
@@ -22,6 +23,7 @@ export const AppRoutes = () => {
   const token = authStorage.getAccessToken();
   const dispatch = useAppDispatch();
   const isAuthorized = auth.useAuthorized();
+  const { flags } = useLaunchDarkly();
```

Comment:
Not blocking/know this is because of how the files are currently written, but would prefer to update `dashboardRoutes` to be `<DashboardRoutes />` so that we can do this call where the flag is being used, instead of passing it through here. But then we'd want to update `builderRoutes` too, and `libraryRoutes`...a whole thing. Anyway just speaking my piece haha

Reply:
I initially set it up as `<DashboardRoutes />` however...we're doing conditional rendering in `<AppRoutes />` and react complains about wrong use of hooks